components/configuration/configuration-read-service-impl/src/main/kotlin/net/corda/configuration/read/impl/ConfigReadServiceEventHandler.kt

```diff
@@ -106,9 +106,11 @@ internal class ConfigReadServiceEventHandler(
             }
 
             is RegistrationStatusChangeEvent -> {
-                // Only registration is on the subscription
+                // Only registration is on the subscriptions
```

Comment:
That sounds like there's one registration covering multiple subscriptions
```suggestion
                // Only set LifecycleStatus.UP for coordinator after the config subscription is UP
```